kratos/tests/test_variables.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+class TestVariables(KratosUnittest.TestCase):
+    def test_01_GetSourceVariable(self): # Tests with numbers to force the order
```

Comment:
I don't think this a good idea, the tests should be independent of each other
Otherwise if I change test 1 then I might have to also change tests 2-x

I think I know why you need this, solution proposal in the next comment